airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/manifest.yaml

```diff
@@ -2059,6 +2059,31 @@ definitions:
           - path: ["dataEndTime"]
             value: "{{ format_datetime(stream_slice['end_time'], '%Y-%m-%d') }}"
 
+api_budget:
+  type: HTTPAPIBudget
+  # Each policy here uses a MovingWindowCallRatePolicy with two rates:
+  # one for burst (per-second) and one for steady (per-minute).
```

Comment:
I don't see the steady ones, is it expected? I'm fine with trying this anyway but we might still get rate limited on the steady case

Reply:
yes, will fix it afterwards. I copypasted comments from klaviyo, amazon has a different calculation